apps/chat/src/components/Common/TalkToNotFound.tsx

```diff
@@ -13,14 +15,31 @@ import { SuggestionButton } from './SuggestionButton';
 interface TalkToNotFound {
   isMyWorkspace: boolean;
   onOpenMarketplaceTab: () => void;
+  isSearchMode?: boolean;
 }
 
 export const TalkToNotFound = ({
   isMyWorkspace,
   onOpenMarketplaceTab,
+  isSearchMode,
 }: TalkToNotFound) => {
   const { t } = useTranslation(Translation.Chat);
 
+  if (isSearchMode) {
```

Comment:
search mode when search term is not empty

```suggestion
  if (!isSearchMode) {
```